packages/devtools_app_shared/lib/src/ui/theme/theme.dart

```diff
@@ -599,3 +603,23 @@ ButtonStyle _generateButtonStyle({
   }
   return buttonStyle;
 }
+
+extension on Color {
```

Comment:
I see we already have these utilities in devtools_app: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/common_widgets.dart/#L944

Instead of creating new utility methods here, can we move the existing ones to devtools_app_shared: https://github.com/flutter/devtools/blob/master/packages/devtools_app_shared/lib/src/ui/ui_utils.dart/#L1